.github/workflows/self-assign.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           echo "Assigning issue $ISSUE_NUMBER to $LOGIN"
           echo "Using the link: https://api.github.com/repos/$REPO/issues/$ISSUE_NUMBER/assignees"
           curl -H "Authorization: token $GITHUB_TOKEN" -d '{"assignees":["'"$LOGIN"'"]}' https://api.github.com/repos/$REPO/issues/$ISSUE_NUMBER/assignees
+          curl -X DELETE -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/$REPO/issues/$ISSUE_NUMBER/labels/awaiting%20triage
```

Comment:
Note: This might feel redundant since we're also assigning the issue, which might lead you to expect the other workflow to trigger and do this. That won't happen because GitHub doesn't trigger any events that were initiated by the github bot itself to infinite trigger loops.